packages/perseus/src/types.ts

```diff
@@ -90,7 +91,7 @@ export type ChangeHandler = (
         // perseus-all-package/widgets/grapher.jsx
         plot?: any;
         // Interactive Graph callback (see legacy: interactive-graph.tsx)
-        graph?: InteractiveGraphState;
+        graph?: PerseusGraphType;
```

Comment:
This type was wrong - InteractiveGraphState is the internal state of our StatefulMafsGraph component - we don't want it to leak outside that component. `PerseusGraphType` is what gets saved in the widget JSON.